util/src/main/java/io/kubernetes/client/util/ClientBuilder.java

```diff
@@ -27,12 +27,9 @@
 import io.kubernetes.client.util.credentials.AccessTokenAuthentication;
 import io.kubernetes.client.util.credentials.Authentication;
 import io.kubernetes.client.util.credentials.KubeconfigAuthentication;
-import java.io.ByteArrayInputStream;
-import java.io.File;
-import java.io.FileReader;
-import java.io.IOException;
-import java.io.Reader;
+import java.io.*;
```

Comment:
expand the wildcard

Reply:
In this class we are using bunch of classes like BufferedReader, InputStreamReader, FileInputStream, File, Reader from java.io package. I can expand it to specify each imported class but it will just add more lines in import section.

Reply:
I have expanded it.